### DISCUSSION TITLE
<title>Unlinking GH Pages from Namecheap custom domain

### DISCUSSION THREAD

**Discussion by DariBerrie** [community/community#40437] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello all!

I accidentally selected the GitHub Pages option when signing up for a free .me domain with the Student Developer Pack.
I don't want my custom domain to point to this page, so I unlinked it per Namecheap's instructions by deleting the DNS A records.

I also deleted the dariberrie.github.io repository that was created with a CNAME file that used my custom domain address.

But several hours later now, it seems like my GitHub Page has disappeared into the ether. When I want to go to dariberrie.github.io/profile OR dariberrie.github.io/profile, this is the result.

<img width="1511" alt="Screen Shot 2022-11-30 at 13 00 01" src="https://user-images.githubusercontent.com/59029920/204791537-2ea2c6c2-ba50-4dff-97e5-19ec396c3485.png">

But when I go to dariberrie.github.io/landing, I go to the page.

<img width="1190" alt="Screen Shot 2022-11-30 at 13 05 19" src="https://user-images.githubusercontent.com/59029920/204791812-b7a2caef-ab2d-440f-ad28-187978588052.png">

I really have no idea what's going on. My dig command gives me this:
```
; <<>> DiG 9.10.6 <<>> www.dariberrie.github.io +noall +answer
;; global options: +cmd
www.dariberrie.github.io. 3600  IN      A       185.199.108.153
www.dariberrie.github.io. 3600  IN      A       185.199.109.153
www.dariberrie.github.io. 3600  IN      A       185.199.110.153
www.dariberrie.github.io. 3600  IN      A       185.199.111.153
```
Which I believe were the A records that I deleted from Namecheap, but I think this is OK? I'm obviously not an expert.

**Comment by DSDmark** [comment, marked as answer]
Hi @DariBerrie,

Did you remove your domain from there?

![Page section inside settings](https://user-images.githubusercontent.com/85744816/204803699-b121abbc-f91c-45c4-a3cc-608057d71397.png)

And also the above image which have that URL ```https://dariberrie.github.io/profile```. It's not GitHub fault. It's because your site don't contain that path ```https://dariberrie.github.io/profile```.

Also, not sure about that. But, when I open your site **URL** look like this.

![image](https://user-images.githubusercontent.com/85744816/204805305-0280e006-d11d-447b-8bf7-040b492d0d20.png)

When you deploy page on **GitHub**. It's look like this, ```https://{your-user-name}.github.io/{your-project-name}``` not like this ```https://dariberrie.github.io```. It's a normal reaction of GitHub page. If you open my like this. You see the same result. https://dsdmark.github.io/. But if you set your project name. Then, you see like this:- https://dsdmark.github.io/BouncingBall

**Reply by DariBerrie** [reply]
That helped me so much @DSDmark ! Knowing that I shouldn't see anything on the https://dariberrie.github.io/ page made something click in my head, so I went back to the repository itself, made a small change and then pushed my code. For w/e reason, it looks like my repository itself was modified, but when I committed the change and pushed to gh-pages, the project showed back up again, but an older version.

Unfortunately, there's now a different problem of the repository not deploying to gh-pages, but I'm really not sure what is causing this.

**Reply by DSDmark** [reply]
Hi @DariBerrie,

> Unfortunately, there's now a different problem of the repository not deploying to gh-pages, but I'm really not sure what is causing this.

Could you describe more about your error which you are facing? And if you can, provide your website source code here. Then, it would help to find out what causing the issue

**Reply by DariBerrie** [reply]
At present, the deployment just doesn't happen. As with your page it should say it's live, but it doesn't even after pushing my gh-pages branch. It's essentially a simple HTML and CSS file that you can view [here](https://github.com/DariBerrie/profile).

<img width="1226" alt="Screen Shot 2022-11-30 at 16 52 56" src="https://user-images.githubusercontent.com/59029920/204848474-c5c1b19d-7ae0-4d29-854b-a7b1bc19ca39.png">

**Comment by AKCCanineHealth** [comment]
On GitHub, navigate to your site's repository.
Under your repository name, click Settings.
In the "Code and automation" section of the sidebar, click Pages.
Under "Custom domain," click Remove[.](https://akcmarketplacepuppies.com/)

**Comment by DSDmark** [comment]
Hi @DariBerrie,

I think you want your site like this, right.

> https://dsdmark.github.io/profile/

```It's a fork repo, once you're done I simply delete this.```

Then, about whole process, to deploy your site like this.

Go to your repo, which link you was provided above. And open the setting of that repo and follow these steps.

https://github.com/orgs/community/discussions/38632#discussioncomment-4092967

And about your that issue.

![image](https://user-images.githubusercontent.com/85744816/204853959-ec8680f6-e9e5-4d70-a063-0f52c9743fda.png)

It's cause because you have that **GitHub page** :- https://dariberrie.github.io/landing/ on that repo:- https://github.com/DariBerrie/landing.

If you have any repo which is already deployed on GitHub page. You can simply open by using that pattern:-```https://{github-user-name}.github.io/{your-project-name}```. (if its deployed).



When you will or was going to deploy your site. Your base URL will always like this :- https://dariberrie.github.io/

And your site which you deploy. Always, Look like this pattern ```https://dariberrie.github.io/{your-project-name}```
